source/stylesheets/mixins/_lists.scss

```diff
@@ -1,6 +1,5 @@
 @mixin plain-list {
-  &,
-  & li {
+  &, & > li {
```

Comment:
Ok, so we don't put each selector on a separate line. This was a style that I picked up at several projects. But I'm fine with dropping it.